[2025-05-01T14:18:03.588+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: training_and_selection.train_random_forest manual__2025-05-01T14:17:58.221362+00:00 [queued]>
[2025-05-01T14:18:03.638+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: training_and_selection.train_random_forest manual__2025-05-01T14:17:58.221362+00:00 [queued]>
[2025-05-01T14:18:03.638+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-01T14:18:03.684+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): train_random_forest> on 2025-05-01 14:17:58.221362+00:00
[2025-05-01T14:18:03.729+0000] {standard_task_runner.py:60} INFO - Started process 249 to run task
[2025-05-01T14:18:03.755+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'training_and_selection', 'train_random_forest', 'manual__2025-05-01T14:17:58.221362+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/training_pipeline_dag.py', '--cfg-path', '/tmp/tmptmt7s73k']
[2025-05-01T14:18:03.760+0000] {standard_task_runner.py:88} INFO - Job 46: Subtask train_random_forest
[2025-05-01T14:18:03.844+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-01T14:18:04.954+0000] {task_command.py:423} INFO - Running <TaskInstance: training_and_selection.train_random_forest manual__2025-05-01T14:17:58.221362+00:00 [running]> on host f4e64b717c7c
[2025-05-01T14:18:05.284+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='training_and_selection' AIRFLOW_CTX_TASK_ID='train_random_forest' AIRFLOW_CTX_EXECUTION_DATE='2025-05-01T14:17:58.221362+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-01T14:17:58.221362+00:00'
[2025-05-01T14:18:05.292+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-05-01T14:18:05.293+0000] {base.py:83} INFO - Using connection ID 'postgres_default' for task execution.
[2025-05-01T14:18:08.253+0000] {logging_mixin.py:188} WARNING - 2025/05/01 14:18:08 INFO mlflow.tracking.fluent: Experiment with name 'diabetes_readmission' does not exist. Creating a new experiment.
[2025-05-01T14:18:08.388+0000] {logging_mixin.py:188} WARNING - 2025/05/01 14:18:08 WARNING mlflow.utils.git_utils: Failed to import Git (the Git executable is probably not on your PATH), so Git SHA is not available. Error: Failed to initialize: Bad git executable.
The git executable must be specified in one of the following ways:
    - be included in your $PATH
    - be set via $GIT_PYTHON_GIT_EXECUTABLE
    - explicitly set via git.refresh(<full-path-to-git-executable>)

All git commands will error until this is rectified.

This initial message can be silenced or aggravated in the future by setting the
$GIT_PYTHON_REFRESH environment variable. Use one of the following values:
    - quiet|q|silence|s|silent|none|n|0: for no message or exception
    - warn|w|warning|log|l|1: for a warning message (logging level CRITICAL, displayed by default)
    - error|e|exception|raise|r|2: for a raised exception

Example:
    export GIT_PYTHON_REFRESH=quiet
[2025-05-01T14:19:03.418+0000] {logging_mixin.py:188} WARNING - 2025/05/01 14:19:03 WARNING mlflow.models.model: Model logged without a signature and input example. Please set `input_example` parameter when logging the model to auto infer the model signature.
[2025-05-01T14:19:05.915+0000] {logging_mixin.py:188} WARNING - Successfully registered model 'diabetes_model'.
[2025-05-01T14:19:05.951+0000] {logging_mixin.py:188} WARNING - Created version '1' of model 'diabetes_model'.
[2025-05-01T14:19:06.271+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-01T14:19:06.326+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=training_and_selection, task_id=train_random_forest, execution_date=20250501T141758, start_date=20250501T141803, end_date=20250501T141906
[2025-05-01T14:19:06.740+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-01T14:19:06.781+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
